src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface TodoItemProps {
+  todo: Todo,
+  loadingTodosId: number[],
+  handleDeleteTodo: (todoId: number) => void
+}
+
+export const TodoItem: React.FC<TodoItemProps> = ({
+  todo,
+  loadingTodosId,
+  handleDeleteTodo,
+}) => {
+  return (
+    <div
+      key={todo.id}
```

Comment:
Don't use the key attribute here. Use it for each item right inside the map() method
```
<TodoItem key={...

Reply:
done